lib/streamlit/ReportThread.py

```diff
@@ -16,6 +16,7 @@
 import threading
 from collections import namedtuple
 
+import streamlit
```

Comment:
Doesn't this cause a circular dependency problem in Python 2?

Usually we have to put circular dependencies at the bottom of the file for Py2: https://stackoverflow.com/a/894885

But if it's not happening here, then OK!

Reply:
It's not, but I'll move it to the bottom of the file to prevent potential future heartache anyway!